Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 1.2.0
+VERSION ?= latest
```

Comment:
Hi @jensfr can you please change this to 1.3.0 as well. My mistake in suggesting `latest`. Really sorry for the multiple iterations :-(